src/GitHub/Data/Webhooks/Payload.hs

```diff
@@ -64,7 +64,7 @@ instance FromJSON OwnerType where
           "organization"  -> pure OwnerOrganization
           _               -> fail $ "Unknown owner type: " ++ T.unpack t
 
-
```

Comment:
Please preserve the white space spacing here, create new white space for documentation rather than inserting into existing space. i.e. there should be no red lines in this diff